ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -870,6 +870,22 @@ AHU_DFSS_DXZC_ECOND_ZHM:
   - ECOND
   - ZHM
 
+AHU_DFSS_CHWDT_STM_RTC_RWISOVPC_OAFC:
+  description: "Smart AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - CHWDT
+  - STM
```

Comment:
DTM, not STM. you use DFSS and CHWDT above.

Reply:
Modified to DTM